### DISCUSSION TITLE
MiniMessage Suggestion: `if` condition

### DISCUSSION THREAD

**Discussion by zefir-git** [KyoriPowered/adventure#1026] [Ideas]
I love using MiniMessage in messages configuration. Sometimes I have to create multiple similar messages that could be avoided with something like this:

```xml
<if:condition:true:false>

You have <if:<has_unread_messages>:<unread_messages>:no unread> messages.
```

If `condition` (preferably a placeholder) is `true` or `1` then the first value should be used, otherwise the 2nd.

The syntax details of course can be changed like using `?` after the condition to make it look exactly like ternary. I figured though `:` is the standard in MiniMessage.

**Comment by kezz** [comment]
We already have something similar to this built in already with the [choice dynamic placeholder](https://docs.advntr.dev/minimessage/dynamic-replacements.html#insert-a-choice), would that not work for you in this situation? We could introduce a standalone `if` tag, but I fail to see where it would be useful as in your case you could just make a choice placeholder.
